.travis.yml

```diff
@@ -20,10 +20,10 @@ jobs:
   include:
     - stage: test
       env:
-        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT="1"
+        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT="1" EXCLUDE="sale_quotation_number"
```

Comment:
is this relevant? It's only for generating POT no?

Reply:
It makes both things: test with `odoo/odoo` and generate the POT, but anyways, you need to generate POT for all modules, so each EXCLUDE/INCLUDE should have both lines.

Reply:
right, fixed